test/utils/Utils.ts

```diff
@@ -79,6 +79,12 @@ export function getRevertMessage(e:Error): string {
   }
   let msgStart = e.message.indexOf('\'');
   if (msgStart !== -1) {
+    let braceIdx = e.message.indexOf("(");
+
+    if (braceIdx !== -1) {
+      return e.message.substring(msgStart + 1, braceIdx);
```

Comment:
Hm, I wonder ehetger this helper should leave this without any special marker. Currently this eould consider `revert("ZeroAddress");` and `revert ZeroAddress;`

Reply:
I used this string manipulation, because in tests I don't always have access to the error arguments and it is hard to validate the error name and arguments, so this way I validate just the error name.

Reply:
I'd probably add a leading character which is not possible for custom errors, such as `:`, to make sure they are never mixed.

```suggestion
      return ":" + e.message.substring(msgStart + 1, braceIdx);
```

Reply:
@StefanVitor @yuval-weiss @mijovic @RedFox20 wdyt?

Reply:
IMO having the same error message as the custom error name is a rare scenario which should be stopped at code review and shouldn't need testing. I added the ":" anyway and will commit it, I am fine with both, can remove it.

Reply:
I don't have a preference on this, looks good either way